derive_builder/src/options/mod.rs

```diff
@@ -233,14 +274,12 @@ impl<Mode> OptionsBuilder<Mode> where
             match *x {
                 syn::NestedMetaItem::MetaItem(ref meta_item) => {
                     self.parse_setter_options_metaItem(meta_item);
-                    // setters implicitly enabled
-                    if self.setter_enabled.is_none() {
-                        self.setter_enabled(true);
-                    }
                 },
-                syn::NestedMetaItem::Literal(ref _lit) => {
-                    // setters explicitly enabled
-                    self.setter_enabled(true);
+                syn::NestedMetaItem::Literal(ref lit) => {
+                    error!("Expected NestedMetaItem::MetaItem, found `{:?}`.", x);
```

Comment:
No `error!` is not user visible, like `trace!` and `warn!` etc. You have to `RUST_LOG=derive_builder=trace cargo test` in order to see this on `stdout`. I consider it debug output. :-)

Btw. just in this moment I am writing the next pull request, which would put all this debug output behind a `logging` feature flag. Since we have unit tests for code generation I found that the logging became far less important. I may even remove it completely one day.